tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1401,3 +1401,59 @@ def test_timeout_and_cac_card_ejection():
 
     :expectedresults: Satellite should terminate the session after mentioned timeout in setting
     """
+
+
+@tier2
+@pytest.mark.skip_if_open("BZ:1670397")
+def test_verify_attribute_of_users_are_updated(session, ldap_data, ldap_tear_down):
+    """Verify if attributes of LDAP user are updated upon first login when
+    onthefly is disabled
+
+    :id: 163b346c-03be-11eb-acb9-0c7a158cbff4
+
+    :Steps:
+        1. Create authsource with onthefly disabled
+        2. Create a user manually and select the authsource created
+        3. Attributes of the user (like names and email) should be synced.
+
+    :BZ: 1670397
+
+    :expectedresults: The attributes should be synced.
+    """
+    ldap_auth_name = gen_string('alphanumeric')
+    auth_source_name = 'LDAP-' + ldap_auth_name
+    with session:
+        session.ldapauthentication.create(
+            {
+                'ldap_server.name': ldap_auth_name,
```

Comment:
I don't see "onthefly" being disabled here. Looks like this test depends on this particular setting default value, and I can't say I'm happy about this.

Could you set that option explicitly here?

Reply:
onthefly is disabled by default, anyway, I mentioned it explicitly.